### PR TITLE
Upgrade Chrome & Edge to v84

### PR DESCRIPTION
**Pull request by iam-mholle**
https://re-motion.atlassian.net/browse/RM-7464

### CODE REVIEW

File: Remotion/Web/Development.WebTesting.IntegrationTests/MouseTest.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@
 using System.Drawing;
 using System.Linq;
 using System.Threading;
+using System.Windows.Forms;
```
Comment (Stefan-Ilic) [inline Remotion/Web/Development.WebTesting.IntegrationTests/MouseTest.cs:22]: Unnecessary using

File: Remotion/Web/Development.WebTesting.IntegrationTests/MouseTest.cs [hunk 2]
```diff
@@ -119,6 +119,7 @@ public void Test_Hover ()
       var home = Start();
 
       var hoverElement = home.Scope.FindId (c_hoverDivID);
+      Thread.Sleep (250);
```
Comment (Stefan-Ilic) [inline Remotion/Web/Development.WebTesting.IntegrationTests/MouseTest.cs:122]: Okay?! If it works, it works. I know how Selenium testing can be 👯

Comment (MichaelKetting) [reply Remotion/Web/Development.WebTesting.IntegrationTests/MouseTest.cs:122]: so sad. of course, we also do this in line 124... Never use mouse, I guess.

**Comment by iam-mholle** [comment]
Added to master in c64fdd9bccde483211ae15271c2c2bd69c95aa82.

**Pull request closed without merging**